Source/Codecov/Coverage/Tool/Coverage.cs

```diff
@@ -21,15 +22,45 @@ public Coverage(ICoverageOptions coverageOptions)
 
         private ICoverageOptions CoverageOptions { get; }
 
-        private static bool VerifyReportFile(string path)
+        private static bool VerifyReportFile(string path, out IEnumerable<string> expandedPath)
```

Comment:
Not sure if I agree with using an out parameter here.
The method should only verify the files, not actually get the files.

Reply:
How about giving it a different name?

Reply:
I was thinking more of separating the globbing logic into a different method that would handle the expanding of the correct file that will be matched.

But I can look into this at a later time.